src/components/profile.ts

```diff
@@ -0,0 +1,20 @@
+import { Database } from 'sqlite';
+export const initUserProfileTable = async (db: Database): Promise<void> => {
```

Comment:
I think @Picowchew recently moved all the init table stuff to `db.ts`, so we should follow that convention too